src/components/common/modals/Confirmation.js

```diff
@@ -130,4 +118,7 @@ Confirmation.propTypes = {
   async: PropTypes.object
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(translate("Confirmation")(Confirmation));
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(translate("Confirmation")(Confirmation));
```

Comment:
Wrong syntax.
Change to translate("Confirmation")(connect(mapStateToProps,mapDispatchToProps)(Confirmation))